docs/remote-facilitator.md

```diff
@@ -0,0 +1,34 @@
+## Overview
+
+The following section explains how remote facilitators for GHO operate and outlines the contracts involved in the setup.
+
+### [OwnableFacilitator](/src/contracts/facilitators/gsm/OwnableFacilitator.sol)
+
+A minimal facilitator contract with minting and burning capabilities, controlled by a single entity. It must be granted the `Facilitator` role on the GHO token with a defined bucket capacity.
+
+This contract can represent a remote minting strategy: it can mint any amount of GHO liquidity on Ethereum, which is then bridged to a remote chain for future use.
+
+### [GhoReserve](/src/contracts/facilitators/gsm/GhoReserve.sol)
+
+It holds GHO liquidity and allows authorized entities to use it up to a defined limit. It is intended for use on remote chains, where GHO is bridged from Ethereum, locked in this reserve, and later distributed by authorized entities through various strategies.
+
+Instead of granting facilitator rights directly on the GHO token to remote facilitators, they are registered as entities in the `GhoReserve`. This indirection is necessary because GHO can only be minted on Ethereum.
+
+A single `GhoReserve` instance can be shared across multiple remote faciltiators, or each remote facilitator can have a dedicated one. Both setups are functionally equivalent, though the shared approach is simpler.
+
+### Remote Facilitators
+
+Remote facilitators function similarly to those on Ethereum, but the way GHO is brought into circulation differs—since GHO can only be minted on Ethereum. Existing minting strategies such as `GhoDirectMinter` (for distribution via Aave V3 Pools) and GSM (for swapping GHO with exogenous assets) can be adapted as remote facilitators using the following setup:
```

Comment:
```suggestion
Remote facilitators function similarly to those on Ethereum, but the way GHO is brought into circulation differs (since GHO can only be minted on Ethereum). Existing minting strategies such as `GhoDirectMinter` (for distribution via Aave V3 Pools) and GSM (for swapping GHO with exogenous assets) can be adapted as remote facilitators using the following setup:
```